vuu-ui/packages/vuu-utils/src/logging-utils.ts

```diff
@@ -9,68 +11,111 @@ export interface TableLogFn {
 	(properties?: object): void
 }
 
+type loggingSettings = {
+	loggingLevel: string;
+}
+
 export interface Logger {
-	log: LogFn;
 	warn: LogFn;
 	error: LogFn;
-	group: LogFn;
-	groupCollapsed: LogFn;
-	groupEnd: LogFn;
-	assert: AssertLogFn;
-	trace: LogFn;
 	debug: LogFn;
 	info: LogFn;
-	table: TableLogFn
 }
 
+export type LogLevel = 'debug' | 'info' | 'warn' | 'error';
+
 export type BuildEnv = 'production' | 'development';
-// eslint-disable-next-line @typescript-eslint/no-empty-function
-const NO_OP: LogFn = () => {};
+
+const NO_OP = () => undefined;
+
+export const loggerNew = (category:string, logLevel: string) => {
```

Comment:
I don't think the logLevel will vary across the app ? Thus no point baking it into each call to logger - might as well read it once inside the logging-utils module